src/main.rs

```diff
@@ -1,21 +1,50 @@
 extern crate docopt;
 extern crate xml;
+extern crate toml;
 
-mod library;
-mod parser;
-
+use std::fs::File;
+use std::io::prelude::*;
 use docopt::Docopt;
 use library::*;
 
+mod library;
+mod parser;
+
 static USAGE: &'static str = "
-Usage: spore -d <dir> <lib>
+Usage: gir [-d <girs_dir>] [<gir>]
+
+Options:
+    -d PATH            Directory for girs
 ";
 
 fn main() {
     let args = Docopt::new(USAGE).unwrap()
         .parse().unwrap_or_else(|e| e.exit());
+    //println!("{:?}", args);
+    //println!("Dir: {}", args.get_str("-d"));
+    //println!("File: {}", args.get_str("<gir>"));
+    
+    let toml = read_toml("Gir.toml");
+    // println!("Toml: {:?}", toml);
+    
+    let girs_dir = match args.get_str("-d") {
+        "" => toml.lookup("options.girs_dir")
+                  .unwrap_or_else(|| panic!("No options.girs_dir in config"))
+                  .as_str().unwrap(),
+        a => a
+    };
+    //println!("Dir: {:?}", girs_dir);
+
+    let gir = match args.get_str("<gir>") {
```

Comment:
The value of this argument is the name of a specific library (Gtk, Gdk, etc) we want to work with.